upup/pkg/fi/cloudup/tests/bootstrapchannelbuilder/amazonvpc/cluster.yaml

```diff
@@ -27,7 +27,10 @@ spec:
   - proxy.api.minimal.example.com
   networkCIDR: 172.20.0.0/16
   networking:
-    amazonvpc: {}
+    amazonvpc:
+      env:
+      - name: WARM_IP_TARGET
+        value: "10"
```

Comment:
could you also show a test with an additional variable?

Reply:
done 👍